src/monattack.cpp

```diff
@@ -5283,6 +5284,41 @@ bool mattack::stretch_attack( monster *z )
     return true;
 }
 
+bool mattack::zombie_fuse( monster *z )
+{
+    const mfaction_id zombo = mfaction_id( "zombie" );
+    const tripoint zpos = z->pos();
+    Creature *critter = nullptr;
+    for( int x = zpos.x - 1; x < zpos.x + 2; x++ ) {
+        for( int y = zpos.y - 1; y < zpos.y + 2; y++ ) {
+            critter = g->critter_at( tripoint( x, y, zpos.z ) );
```

Comment:
```suggestion
            critter = g->critter_at<monster>( tripoint( x, y, zpos.z ) );
```
So you don't need to cast it. And change `critter` to be `monster *`.